generators/authentication/templates/config/passport.js

```diff
@@ -151,6 +151,7 @@ module.exports = {
      */
     onPassportAuth(req, res, error, user, info) {
       if (error || !user) return res.negotiate(error || info);
+      if (info.name === 'TokenExpiredError') info.status = 401; return res.negotiate(info)
```

Comment:
Looks like I can set `expiresInMinutes: 1` for testing the expiration a lot quicker.